doc/source/installation/compiling_xtradb_cluster.rst

```diff
@@ -1,41 +1,92 @@
-===========================================
- Compiling and Installing from Source Code
-===========================================
-
-The source code is available from the *Launchpad* project `here <https://launchpad.net/percona-xtradb-cluster>`_. The easiest way to get the code is with :command:`bzr branch` of the desired release, such as the following: ::
-
-  bzr branch lp:percona-xtradb-cluster/5.6
-
-You should then have a directory named after the release you branched, such as ``percona-xtradb-cluster``.
-
-
-Compiling on Linux
-==================
-
-Prerequisites
--------------
-
-The following packages and tools must be installed to compile *Percona XtraDB Cluster* from source. These might vary from system to system.
-
-In Debian-based distributions, you need to: ::
-
-  $ apt-get install build-essential flex bison automake autoconf bzr \
-    libtool cmake libaio-dev mysql-client libncurses-dev zlib1g-dev
-
-In ``RPM``-based distributions, you need to: ::
-
-  $ yum install cmake gcc gcc-c++ libaio libaio-devel automake autoconf bzr \
-    bison libtool ncurses5-devel
-
-Compiling 
-------------
-
-The most esiest way to build binaries is to run script: ::
-
-  BUILD/compile-pentium64-wsrep
-
-If you feel confident to use cmake, you make compile with cmake adding
--DWITH_WSREP=1 to parameters.
-
-Examples how to build RPM and DEB packages you can find in packaging/percona directory in the source code.
+.. _compile:
+
+=========================================
+Compiling and Installing from Source Code
+=========================================
+
+If you want to compile Percona XtraDB Cluster, you can find the source code on GitHub. Before you begin, make sure that the following packages are installed:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - 
+     - apt
+     - yum
+   * - Git
+     - ``git``
+     - ``git``
+   * - SCons
+     - ``scons``
+     - ``scons``
+   * - GCC
+     - ``gcc``
+     - ``gcc``
+   * - g++
+     - ``g++``
+     - ``gcc-c++``
+   * - OpenSSL
+     - ``openssl``
+     - ``openssl``
+   * - Check
+     - ``check``
+     - ``check``
+   * - CMake
+     - ``cmake``
+     - ``cmake``
+   * - Bison
+     - ``bison``
+     - ``bison``
+   * - Boost
+     - ``libboost-all-dev``
+     - ``boost-devel``
+   * - Asio
+     - ``libasio-dev``
+     - ``asio-devel``
+   * - Async I/O
+     - ``libaio-dev``
+     - ``libaio-devel``
+   * - ncurses
+     - ``libncurses5-dev``
+     - ``ncurses-devel``
+   * - Readline
+     - ``libreadline-dev``
+     - ``readline-devel``
+   * - PAM
+     - ``libpam-dev``
+     - ``pam-devel``
+
+You will likely have all or most of the packages already installed. If you are not sure, run one of the following commands to install any missing dependencies:
+
+.. code-block:: bash
+
+   $ sudo apt-get install -y git scons gcc g++ openssl check bison libboost-all-dev libasio-dev libaio-dev libncurses5-dev libreadline-dev libpam-dev
```

Comment:
Break the lines with "\" otherwise it won't be seen correctly in pdf manual.

sudo apt-get install -y git scons gcc g++ openssl check bison \
libboost-all-dev libasio-dev libaio-dev libncurses5-dev \
libreadline-dev libpam-dev